### ISSUE TITLE
"Add a team member" should accept phone numbers?

### ISSUE THREAD

**Issue by fabacab** [betterangels/buoy#28]
Look up users' phone numbers if any already existed?

What are the concerns for phone number enumeration?

**Comment by fabacab** [comment]
I'm in favor of this feature. I think a lot of people will be more likely to know their contacts' phone numbers than their e-mail addresses. Most of the concerns I can think of are edge cases (e.g. users or robots spamming random phone numbers with team invites.)

Only question I have is how we would notify someone who is not currently a Buoy user that they'd been invited to join a team. Currently, this happens by sending them an e-mail invitation to join Buoy. We could do the same with phone numbers by sending a text message, but this feels possibly more intrusive to me than e-mailing?

**Comment by fabacab** [comment]
This feature might be nice but it opens up a can of worms that we don't have the resources to address right now. For instance:
- We would need to enforce uniqueness on the phone number fields so that one phone number could only be associated with one Buoy account at a given time. WordPress does this for us "for free" with email addresses, but not for phone numbers.
- During sign-up, we would need to perform a phone number verification loop to ensure the person who is actually signing up with a given phone _actually has_ the physical device that they are signing up with. (Again, WordPress does this for us "for free" with email addresses.)
- Using only a txt message interface, there is currently no way to modify one's WordPress user account (profile) details. Doing this requires a Web browser, unless we expand #54 to work beyond Buoy-specific functionality.

In short, while I agree that this is something we should consider adding at some point, I'm putting it on the back burner because it is clearly a project unto itself.
